docs: undeprecate plugin (#54)
apache/cordova-plugin-device-orientation: README.md

# cordova-plugin-device-orientation

# Usage Notice

> With the [W3C Device Orientation API](https://www.w3.org/TR/orientation-event/) now being supported on Android, iOS and Windows devices, this plugin may not needed any more.

Reviewer: ```suggestion
With the [W3C Device Orientation API](https://www.w3.org/TR/orientation-event/), Android, iOS, and Windows devices may not need this plugin anymore.
```

> However, for iOS 13+ a so called [secure context](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts) is required. Apple does not allow access to the device’s orientation or the permission request from insecure browsing contexts. If you use a custom scheme, i.e., do not use the `file` protocol for your iOS app the browsing context is considered insecure. To solve this problem, the plugin is mandatory for iOS 13+ if you plan to use the device orientation feature.

Reviewer: ```suggestion
However, for iOS 13+, a [secure context](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts) is required. Apple does not allow access to the device’s orientation or the permission request from an insecure browsing context.

If you use a custom scheme, i.e., the `app` protocol, for your iOS app, the browsing context is considered insecure. To solve this problem and be able to use the device orientation feature, this plugin will be necessary on iOS 13+.
```
